build/IPDK_Container/scripts/rundemo.sh

```diff
@@ -0,0 +1,232 @@
+#!/bin/bash
+#Copyright (C) 2021 Intel Corporation
+#SPDX-License-Identifier: Apache-2.0
+
+stty -echoctl # hide ctrl-c
+
+# Exit function
+exit_function()
+{
+    echo "Exiting cleanly"
+    pushd /root || exit
+    rm -f network-config-v1.yaml meta-data user-data
+    rm -rf /tmp/vhost-user-*
+    rm -f vm1.qcow2 vm2.qcow2
+    popd || exit
+}
+
+trap 'exit_function' SIGINT
+
+echo ""
+echo "Cleaning from previous run"
+echo ""
+
+rm -rf /tmp/vhost-user-*
+killall ovsdb-server
+killall ovs-vswitchd
+
+#echo ""
+#echo "Pulling down cirros image and configuring"
+#echo ""
+#
+#pushd /root || exit
+#wget -nc http://download.cirros-cloud.net/0.5.2/cirros-0.5.2-x86_64-disk.img
+#cp cirros-0.5.2-x86_64-disk.img vm1.qcow2
+#cp cirros-0.5.2-x86_64-disk.img vm2.qcow2
+#popd || exit
+
+echo ""
+echo "Creating Ubuntu focal image"
+echo ""
+
+pushd /root || exit
+/git/ipdk/scripts/get-image.sh focal
+qemu-img create -f qcow2 -b focal-server-cloudimg-amd64.raw disk.img
```

Comment:
I don't understand... the get-image script you added downloads a ubuntu qcow2 file and converts it to raw and directly after using the script the file is converted back to qcow2 again?

Reply:
Good catch! I'll remove the steps to move back and forth from raw.